### PR TITLE
Hamza Jawed

### PR DESCRIPTION
**Pull request by HamzaJawed-HJ**

### CODE REVIEW

**Review by HamzaJawed-HJ — COMMENTED** [review]
Hamza Jawed

**Comment by hamzaJawed19** [comment]
Hamza Jawed

File: week_six/quiz_class.dart [hunk 1]
```diff
@@ -15,6 +15,12 @@ void main() {
...
-
-  return [2, 5];
+for(int i=0;i<numbers.length;i++){
+     for(int j=i+1;j<numbers.length;j++){
+       if(numbers[i]+numbers[j]==target){
+         return [numbers[i],numbers[j]];
+       }       
+     }
+   }
+  return [];
```
Comment (Muzammil-Bilwani) [inline week_six/quiz_class.dart:25]: Code Formatting can be a good thing. 
Nice work.

**Comment by Muzammil-Bilwani** [comment]
Closing, marking as correct.

**Pull request closed without merging**